Razer Lancehead Tournament Edition (Wired)

aelkz opened the pull request:
Manufacturer: Razer
Device Class: 00(>ifc )
Vendor Id: 1532
Product Id: 0060
Number of Interfaces: 3

aelkz committed:
[UPDATE] Razer Lancehead Tournament Edition
Manufacturer: Razer
Device Class: 00(>ifc )
Vendor Id: 1532
Product Id: 0060
Number of Interfaces: 3

z3ntu commented on driver/razermouse_driver.c:
```diff
@@ -2453,6 +2486,7 @@ static const struct hid_device_id razer_devices[] = {
     { HID_USB_DEVICE(USB_VENDOR_ID_RAZER,USB_DEVICE_ID_RAZER_MAMBA_2012_WIRELESS) },
     { HID_USB_DEVICE(USB_VENDOR_ID_RAZER,USB_DEVICE_ID_RAZER_MAMBA_WIRED) },
     { HID_USB_DEVICE(USB_VENDOR_ID_RAZER,USB_DEVICE_ID_RAZER_MAMBA_WIRELESS) },
+    { HID_USB_DEVICE(USB_VENDOR_ID_RAZER,USB_DEVICE_ID_LANCEHEAD_TE_WIRED) },
```
Should be `USB_DEVICE_ID_RAZER_LANCEHEAD_TE_WIRED`

aelkz replied:
fixed. thanks

z3ntu commented on daemon/openrazer_daemon/hardware/mouse.py:
```diff
@@ -93,6 +93,28 @@ class RazerMambaTE(__RazerDeviceBrightnessSuspend):
+               'get_dpi_xy', 'set_dpi_xy']
+
+    RAZER_URLS = {
+        "top_img": "https://d1urewwzb2qwii.cloudfront.net/sys-master/root/h5e/h0b/8888236245022",
```
Aren't there images available on the Razer site? I'd rather not have externally hosted images.

aelkz replied:
yeah.. tried to look the generated html page and didn't see any reference. Also I have tried several different naming patterns and didn't find any match.. =/
The product ID is: 27310,27309

For example:
https://assets.razerzone.com/eeimages/products/27310/lanceheadte-og-1200x630-v1.png
https://assets.razerzone.com/eeimages/products/27309/ogimage-1200x630.jpg

z3ntu replied:
side_img: https://assets.razerzone.com/eeimages/products/27310/lanceheadte-galleryimg-1500x1000-4.png
perspective_img: https://assets.razerzone.com/eeimages/products/27310/lanceheadte-galleryimg-1500x1000-5.png
Didn't find a nice top_img yet

aelkz committed:
[FIX] Razer Lancehead Tournament Edition
Manufacturer: Razer
Device Class: 00(>ifc )
Vendor Id: 1532
Product Id: 0060
Number of Interfaces: 3

aelkz committed:
[FIX] Razer Lancehead Tournament Edition
Manufacturer: Razer
Device Class: 00(>ifc )
Vendor Id: 1532
Product Id: 0060
Number of Interfaces: 3

z3ntu committed:
Add Lancehead to README

z3ntu committed:
Add Lancehead to udev rules

z3ntu committed:
autopep8 lancehead script

z3ntu committed:
Replace device pictures with ones on a razer subdomain

aelkz committed:
[FIX] Razer Lancehead Tournament Edition
Manufacturer: Razer
Device Class: 00(>ifc )
Vendor Id: 1532
Product Id: 0060
Number of Interfaces: 3

aelkz committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	README.md
#	daemon/openrazer_daemon/hardware/mouse.py

aelkz committed:
[FIX] Razer Lancehead Tournament Edition
Manufacturer: Razer
Device Class: 00(>ifc )
Vendor Id: 1532
Product Id: 0060
Number of Interfaces: 3

aelkz committed:
[UPDATE]
Trying to clean a little and test with minimal enabled features for Lancehead TE

aelkz committed:
[UPDATE]
Trying to clean a little and test with minimal enabled features for Lancehead TE

aelkz committed:
[FIX]
USB_DEVICE_ID_RAZER_ABYSSUS_1800 missing.

aelkz committed:
[FIX]
Razer LanceheadTE class adjustments

aelkz committed:
[FIX]
Razer LanceheadTE class adjustments

aelkz committed:
[FIX]
Razer LanceheadTE class adjustments
